spec/support/bulk_upload/lettings_log_to_csv.rb

```diff
@@ -12,6 +12,62 @@ def row_prefix
     [nil] * col_offset
   end
 
+  def to_csv_row(seed: nil)
+    year = log.collection_start_year
+    case year
+    when 2022
```

Comment:
Not sure the 2022 case is really necessary for these, but some of the 2023 functions rely on the 2022 ones so I didn't want to get rid of them - at which point this case may as well be here.